src/System.Linq.Expressions/tests/IndexExpression/IndexExpressionHelpers.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Xunit;
+
+namespace System.Linq.Expressions.Tests.IndexExpression
```

Comment:
For consistency with other tests, this could just be `System.Linq.Expressions.Tests`. It also removes the need for qualification or aliasing of `IndexExpression`.